src/main/web/templates/handlebars/content/partials/intro-meta.handlebars

```diff
@@ -1,4 +1,4 @@
-{{#if description.releaseDate}}
+{{#if_all type "static_page" description.releaseDate}}
```

Comment:
I am not sure this works, possibly looking for type_code "static_page"

Reply:
It's definitely not working on my machine the way I thought it was. Not sure what happened. Making the change.